From f76661a41f7db88c6cbbe1831fddb91601514204 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 07 Aug 2018 11:19:56 +0300
Subject: [PATCH] board: habana: remote_run: remove needless 'if'

Remove needless 'if' statement and thus lower the indentation
and simplify flow.
Also, we do not need to remove the log file each time we have a ping...

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_run b/board/habanalabs/common/remote_run
index 313a57a..1fdd217 100755
--- a/board/habanalabs/common/remote_run
+++ b/board/habanalabs/common/remote_run
@@ -75,15 +75,12 @@
 for ((i=100; i>0; i--)); do
 	echo $i
 	sleep 1;
-	ping -c 1 -W 1 $host
-	if [ $? -eq 0 ]; then
-		rm -f $result
-		remote_prepare_work_dirs	|| continue;
-		remote_copy_fw_scripts		|| break;
-		remote_run_script		|| break;
-		echo done;
-		exit 0;
-	fi
+	ping -c 1 -W 1 $host		|| continue;
+	remote_prepare_work_dirs	|| continue;
+	remote_copy_fw_scripts		|| break;
+	remote_run_script		|| break;
+	echo done;
+	exit 0;
 done
 
 echo error
